### DISCUSSION TITLE
Win32:  How can I add a (nativ) "open file dialog" on axmol ?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1041] [Q&A] [answered]
Win32:  How can I add a (nativ) "OpenFile Dialog" on axmol ?

**Comment by DelinWorks** [comment, marked as answer]
since you're using c++ to develop games and also using a msvc compiler, you can take advantage of the Win32 API calls like MessageBoxW or something like that. I'm sure it's on a Microsoft developer website

Here's an example in c++ that you can embed in your app the way you want, it would be cool if it was integrated in the engine as a helper class
https://learn.microsoft.com/en-us/windows/win32/learnwin32/example--the-open-dialog-box
